my-app/src/components/column.js

```diff
@@ -1,33 +1,57 @@
 import { Button } from "react-bootstrap";
 import { useEffect } from "react";
+import "./column.css";
 import "./body.css";
```

Comment:
delete body.css import from here,
move any neede styles from body.css to column.css